packages/dev/serializers/src/glTF/2.0/glTFExporter.ts

```diff
@@ -1426,7 +1426,7 @@ export class _Exporter {
             }
             const target: { [attribute: string]: number } = {};
             if (babylonMorphTarget.hasNormals) {
-                const vertexNormals = babylonSubMesh.getMesh().getVerticesData(VertexBuffer.NormalKind)!;
+                const vertexNormals = (babylonSubMesh.getMesh() as Mesh).getVerticesData(VertexBuffer.NormalKind, undefined, undefined, true)!;

```

Comment:
Just wondering if we are fully sure it is a mesh we could either pass it in the params ? or have a local mesh variable to use " as Mesh" only once ?